SpatialGDK/Source/SpatialGDKEditor/Private/Utils/SchemaBundleParser.cpp

```diff
@@ -66,8 +66,19 @@ TSharedPtr<FJsonObject> OpenJsonFile(const FString& SchemaJsonPath)
 
 namespace SpatialGDK
 {
+void AddToListIfClearable(const TSharedPtr<FJsonObject>* ArrayObject, int32 FieldId, TArray<uint32>& OutIDs)
+{
+	const TSharedPtr<FJsonObject>* JsonObject;
+	// Only listType's are clearable, for the moment at least.
```

Comment:
I know this is just in-progress, but I would replicate the docs here, saying lists, options, and maps are clearable, but we only currently care about lists since the other types cannot be generated from Unreal properties.